camayoc/tests/qcs/cli/utils.py

```diff
@@ -0,0 +1,34 @@
+# coding=utf-8
+"""Utility functions for Quipucords cli tests."""
+import pexpect
+
+
+def cred_add(options, inputs=None, exitstatus=0):
+    """Add a new credential entry.
+
+    :param options: A dictionary mapping the option names and their values.
+        Pass ``None`` for flag options.
+    :param inputs: A dictionary mapping the input prompts and the value to be
```

Comment:
It looked like at [this line](https://github.com/quipucords/camayoc/pull/105/files#diff-da2722cd547bb9c981a963c78353e2c6R45) "inputs" is supposed to be a list?

Can you provide an example in the doc string of usage?

Reply:
Yes it should be a list, thank you